pinot-spi/src/main/java/org/apache/pinot/spi/config/table/TableConfig.java

```diff
@@ -79,6 +80,9 @@
   @JsonPropertyDescription(value = "upsert related config")
   private UpsertConfig _upsertConfig;
 
+  @JsonPropertyDescription(value = "Config related to table ingestion")
```

Comment:
```suggestion
  @JsonPropertyDescription(value = "Config related to ingesting data into the table")
```

Should we consider moving append frequency into this?

Reply:
Yes that's a good one to go into this config

Reply:
Noted in the design doc extensions section, where more such are listed